composites/Plugin/ContentAnalysis/components/AnalysisCollapsible.js

```diff
@@ -0,0 +1,163 @@
+import PropTypes from "prop-types";
+import React from "react";
+import styled from "styled-components";
+import { angleUp, angleDown } from "../../../../style-guide/svg";
+import colors from "../../../../style-guide/colors.json";
+import { IconButton } from "../../Shared/components/Button";
+
+/**
+ * Container for the Collapsible header and its content.
+ */
+const AnalysisHeaderContainer = styled.div`
+	margin: 8px 0;
+	background-color: ${ colors.$color_white };
+`;
+
+/**
+ * The clickable component of the analysis header.
+ */
+const AnalysisHeaderButton = styled( IconButton )`
+	width: 100%;
+	background-color: ${ colors.$color_white };
+	padding: 0;
+	border-color: transparent;
+	border-radius: 0;
+	outline: none;
+	box-shadow: none;
+	// When clicking, the button text disappears in Safari 10 because of color: activebuttontext.
+	color: ${ colors.$color_blue };
+
+	:hover {
+		border-color: transparent;
+	}
+
+	:active {
+		box-shadow: none;
+		background-color: ${ colors.$color_white };
+	}
+
+	svg {
+		margin: 0 10px;
+		width: 20px;
+		height: 20px;
+	}
+`;
+
+/**
+ * The analysis header text.
+ */
+const AnalysisTitle = styled.span`
+	margin: 8px 0;
+	word-wrap: break-word;
+	font-size: 1.25em;
+	line-height: 1.25;
+`;
+
+/**
+ * Analysis items list.
+ */
+const AnalysisList = styled.ul`
+	margin: 0;
+	list-style: none;
+	padding: 8px 16px;
+`;
+
+/**
+ * A collapsible header used to show sets of analysis results. Expects list items as children.
+ *
+ * @param {object} props The properties for the component.
+ * @returns {ReactElement} A collapsible analysisresult set.
+ */
+export const AnalysisCollapsibleStateless = ( props ) => {
+	return (
+		<AnalysisHeaderContainer>
+			<AnalysisHeaderButton
+				aria-expanded={ props.isOpen }
+				onClick={ props.onToggle }
+				icon={ props.isOpen ? angleUp : angleDown }
+				iconColor={ colors.$color_grey_dark }
+			>
+				<AnalysisTitle>{ props.title + " (" + props.children.length + ")" }</AnalysisTitle>
```

Comment:
We shouldn't assume there are always children with a length:
- when there are no children, this will error because `children` is undefined: `TypeError: Cannot read property 'length' of undefined`
- when there's just 1 problem, `children` is not an array: it's a node (react.element) so it doesn't have a length property